tests/Process/StateChangeCallbackTest.php

```diff
@@ -6,20 +6,30 @@
 use PHPSu\Process\Process;
 use PHPSu\Process\ProcessManager;
 use PHPSu\Process\StateChangeCallback;
+use PHPSu\Tools\EnvironmentUtility;
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Console\Formatter\OutputFormatter;
 use Symfony\Component\Console\Output\BufferedOutput;
-use Symfony\Component\Console\Output\ConsoleSectionOutput;
 use Symfony\Component\Console\Output\OutputInterface;
 
+if (!class_exists('ConsoleSectionOutput', false)) {
```

Comment:
What do we need this check for?
``if (!class_exists('ConsoleSectionOutput', false)) {``